ui/app/pages/swaps/swaps.util.js

```diff
@@ -323,6 +327,21 @@ export async function fetchSwapsFeatureLiveness() {
   return status?.active
 }
 
+export async function fetchSwapsQuoteRefreshTime(defaultValue = 0) {
+  const response = await fetchWithCache(
+    getBaseApi('refreshTime'),
+    { method: 'GET' },
+    { cacheRefreshTime: 600000 },
+  )
+
+  // We presently use milliseconds in the UI
+  if (response?.seconds) {
+    return response.seconds * 1000
+  }
+
+  return defaultValue
```

Comment:
I'm not sure about this solution :thinking: If the API started returning a malformed response, this would be a silent failure, and we'd never know the difference. The app would continue to use the default. Throwing an error might be safer.